### PR TITLE
Variables and data types

### PR DESCRIPTION
**Pull request by danacebryantalfonso**
Changed the folder name and added the exercise 2 and 3.

### CODE REVIEW

File: Strings and Data Types/src/StringsAndFunctions.kt [hunk 1]
```diff
@@ -1,21 +1,81 @@
...
+    varPrint.variablesAndStrings()
+    varPrint.discoverNewFunction()
+    varPrint.StringsExercise()
 
-    println(myPet.get(3))
-    println(myPet[3])
-    println(myPet.substring(3))
-    println(myPet.substring(3,6))
+}
+class StringsAndDataTypes:Exercises(){
```
Comment (rexendy-lig) [inline Strings and Data Types/src/StringsAndFunctions.kt]: apply spacing
ex.
class StringsAndDataTypes : Exercises() {

File: Strings and Variables/.idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (rexendy-lig) [inline Strings and Variables/.idea/.gitignore]: apply necessary classes/folders in .gitignore
ex.
include .idea in .gitingore

**Review by rexendy-lig — CHANGES_REQUESTED** [review]

File: Strings and Data Types/src/StringsAndFunctions.kt (hunk 1 above)
Comment (danacebryantalfonso) [reply Strings and Data Types/src/StringsAndFunctions.kt]: Fixed this in d4a2890da6f59994e6561d5e61d373d29eee2be0

File: Strings and Variables/src/Exercise1.kt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+fun main(args: Array<String>){
+//    val catName = "My cat's name is \"Fluff\""
+//    println(catName)
+//
+//    val slashes = "Two types of slashes: \\ and /"
+//    println(slashes)
+//
+//    val url = "http://www.google.com"
+//    println(url)
```
[the remark this answers is not in the document]
Comment (danacebryantalfonso) [reply Strings and Variables/src/Exercise1.kt]: comment removed in c578ba0a0cd5f71366ca440d9af87f911997d170

File: Strings and Data Types/src/StringsAndFunctions.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (danacebryantalfonso) [reply Strings and Data Types/src/StringsAndFunctions.kt]: Added spaces after closing braces in c578ba0a0cd5f71366ca440d9af87f911997d170

File: Strings and Variables/.idea/.gitignore (hunk 2 above)
Comment (danacebryantalfonso) [reply Strings and Variables/.idea/.gitignore]: Added those folders and files in 07953db5071a9a2074d902bae929c099b1638636

**Review by rexendy-lig — APPROVED** [review]

**Pull request merged**